src/main/java/com/synopsys/integration/jira/common/model/response/IssueCreatemetaFieldResponseModel.java

```diff
@@ -0,0 +1,88 @@
+/**
+ * int-jira-common
+ *
+ * Copyright (c) 2020 Synopsys, Inc.
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership. The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package com.synopsys.integration.jira.common.model.response;
+
+import java.util.List;
+
+import com.google.gson.JsonElement;
+import com.synopsys.integration.jira.common.model.JiraResponseModel;
+import com.synopsys.integration.jira.common.model.components.SchemaComponent;
+
+public class IssueCreatemetaFieldResponseModel extends JiraResponseModel {
```

Comment:
should "Meta" be capitalized?

Reply:
Weridly they did not capitalize it in their API https://developer.atlassian.com/cloud/jira/platform/rest/v2/api-group-issues/#api-rest-api-2-issue-createmeta-get